docs/asciidoc/gettingStarted.adoc

```diff
@@ -83,20 +83,25 @@ However, if you want to force the use of a specific artifact's version, you can
 it when adding your dependency, as you usually would. You can also forgo the BOM entirely
 and specify dependencies by their artifact versions.
 
+WARNING: As of this version (reactor-core {project-version}), the latest stable BOM in the associated
+release train line is `{reactorReleaseTrain}`, which is what is used in snippets below.
+There might be newer versions since then (including snapshots, milestones and new release train lines),
+see https://github.com/reactor/reactor/releases/.
```

Comment:
wouldn't it be better to link to https://projectreactor.io/docs that contains both the release train + core's version?

Reply:
indeed, fixed